Update README.md (#12715)
fleetdm/fleet: docs/Using-Fleet/configuration-files/README.md

webhook_settings
```

> You can bypass these errors by removing the key from your YAML or adding the `--force` flag. This flag will force the application of the changes without validation but should be used with caution.

Reviewer: Tiny thing but I'd use "and" instead of "but" here. Also, I think we can chop some more words 🔪 😇 
```suggestion
You can bypass these errors by removing the key from your YAML or adding the `--force` flag. This flag will apply the changes without validation and should be used with caution.
```

You can bypass these errors by removing the key from your YAML or adding the `--force` flag. This flag will force the application of the changes without validation but should be used with caution.

### Mobile device management (MDM) settings for teams

> > MDM features are not ready for production and are currently in development. These features are disabled by default.

Author: @rachaelshaw, although the features are perhaps still disabled by default?

Reviewer: @mike-j-thomas I'm not sure — requesting a review from @noahtalerman to clarify
